vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/StaffSearchManager.java

```diff
@@ -0,0 +1,51 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.diging.vspace.core.data.TextContentBlockRepository;
+import edu.asu.diging.vspace.core.model.IVSpaceElement;
+import edu.asu.diging.vspace.core.services.IModuleManager;
+import edu.asu.diging.vspace.core.services.ISlideManager;
+import edu.asu.diging.vspace.core.services.ISpaceManager;
+import edu.asu.diging.vspace.core.services.IStaffSearchManager;
+
+@Service
+public class StaffSearchManager implements IStaffSearchManager{
+
+    @Autowired
+    private ISpaceManager spaceManager;
+    
+    @Autowired
+    private IModuleManager moduleManager;
+    
+    @Autowired
+    private ISlideManager slideManager;
+    
+    @Autowired
+    private TextContentBlockRepository textContentBlockRepo;
+    
+    
+    /**
+     * This method is used to fetch the staff side search. The search text searched 
+     * in space, module and slide tables.
```

Comment:
"fetch the staff side search" does not sound right and "search text searched in" is also not proper English. Why not "... used to search the given text in the name and description field of... "